test/verify/check-networkmanager-basic

```diff
@@ -412,14 +412,17 @@ class TestNetworkingBasic(netlib.NetworkCase):
         b.wait_visible("#network-ip-settings-address-0")
         # test class A IP
         b.set_input_text("#network-ip-settings-address-0", "10.0.5.1")
-        b.wait_val("#network-ip-settings-netmask-0", "255.0.0.0")
+        b.wait_val("#network-ip-settings-netmask-0", "8")
+        b.set_input_text("#network-ip-settings-netmask-0", "");
 
         # test class B IP
         b.set_input_text("#network-ip-settings-address-0", "172.16.44.2")
-        b.wait_val("#network-ip-settings-netmask-0", "255.255.0.0")
```

Comment:
Do we still test netmasks? They are still supported, right?

Reply:
Yes, in case of IPv4 both prefix length and netmask are supported. Tests in `testIpHelper` use both.

```py
        # /23
        b.set_input_text("#network-ip-settings-netmask-0", "255.255.254.0")
        b.wait_val("#network-ip-settings-gateway", "10.11.100.1")

        b.set_input_text("#network-ip-settings-netmask-0", "24")
        b.wait_val("#network-ip-settings-gateway", "10.11.101.1")
```

Reply:
The change in behavior here is that the "default" is to use prefix length over netmask.